main-unix.c

```diff
@@ -308,13 +342,19 @@ main (int argc,
       else
       {
         JERRY_ERROR_MSG ("Error: wrong format of the arguments\n");
+        print_usage ();
         return JERRY_STANDALONE_EXIT_CODE_FAIL;
       }
     }
     else if (!strcmp ("--abort-on-fail", argv[i]))
     {
       flags |= JERRY_FLAG_ABORT_ON_FAIL;
     }
+    else if (!strncmp ("-", argv[i], 1))
+    {
```

Comment:
I think we need an error message here to let the user know why we are exiting. E.g., `"Error: unrecognized option: %s\n", argv[i]` .

BTW, might be worth revisiting the other error messages of arg handling as well and improve them a little. E.g, `"Error: wrong format of the arguments\n"` doesn't say much at `--save-snapshot-for-global`. We could have something like:
```c
if (save_snapshot_file_name_p != NULL)
{
  JERRY_ERROR_MSG ("Error: snapshot file name already specified\n");
  return JERRY_STANDALONE_EXIT_CODE_FAIL
}
if (++i >= argc)
{
  JERRY_ERROR_MSG ("Error: missing argument of %s\n", argv[i-1]);
  return JERRY_STANDALONE_EXIT_CODE_FAIL
}
```

Similarly elsewhere.